include/RAJA/policy/sycl/kernel/SyclKernel.hpp

```diff
@@ -186,7 +161,7 @@ struct SyclLaunchHelperNonTrivial<sycl_launch<async0>,StmtList,Data,Types>
  * determined at runtime using the SYCL occupancy calculator.
  */
 template<bool async0, typename StmtList, typename Data, typename Types>
-struct SyclLaunchHelper<sycl_launch<async0>,StmtList,Data,Types>
+struct SyclLaunchHelper<true,sycl_launch<async0>,StmtList,Data,Types>
```

Comment:
On line 189 below, `async` will always be true, so the queue will never `wait()`. Is this the behavior we want?